loader/merge.go

```diff
@@ -247,6 +250,14 @@ func mergeServiceNetworkConfig(dst, src reflect.Value) error {
 	return nil
 }
 
+//nolint: unparam
+func mergeServiceCommand(dst, src reflect.Value) error {
```

Comment:
Some comment on this code would be useful to understand why it's there (some callback executed by reflection I guess).
Does it need to be added [here](https://github.com/compose-spec/compose-go/blob/master/loader/merge.go#L40) ? otherwise, how does this work ?

Reply:
obsolete sorry, was my first attempt trying to fix this